src/main/scala/dsptools/counters/Counters.scala

```diff
@@ -66,21 +66,21 @@ class CountIO (countParams: CountParams) extends Bundle {
   // Count up/down control signal
   val upDown = if (countParams.countType == UpDown) Some(Input(Bool())) else None
   // Counters usually increment by 1
-  val inc = if (countParams.incMax != 1) Some(Input(UInt(countParams.incMax))) else None
```

Comment:
Actually, sorry. My comment was referring to my original code, with DSPUInt[max]. Seems like when Chick translated it over to Chisel3, it got incorrectly replaced with UInt[Lit]. 

Original being `val inc = if (countParams.incMax != 1) Some(DSPUInt(INPUT,countParams.incMax)) else None`